Casks/c/claude.rb

```diff
@@ -0,0 +1,38 @@
+cask "claude" do
+  version "0.4.8,abfc4944184bdf344113a54b0fcaddfc8040176b"
+  sha256 "6e3cc0dfdb145128790855aee8ed08b76a32e0f01827fddf86a0f27836c4da73"
+
+  url "https://storage.googleapis.com/osprey-downloads-c02f6a0d-347c-492b-a752-3e0651722e97/nest/release-#{version.csv.first}-artifact-#{version.csv.second}.zip",
+      verified: "storage.googleapis.com/osprey-downloads-c02f6a0d-347c-492b-a752-3e0651722e97/nest/"
+  name "Claude"
+  desc "Anthropic's official Claude AI desktop app"
+  homepage "https://claude.ai/download"
+
+  livecheck do
+    url "https://storage.googleapis.com/osprey-downloads-c02f6a0d-347c-492b-a752-3e0651722e97/nest/update_manifest.json"
+    regex(/release[-_.](\d+(?:\.\d+)+)[-_.]artifact[-_.](\h+)\.zip$/i)
```

Comment:
```suggestion
    regex(/release[._-]v?(\d+(?:\.\d+)+)[._-]artifact[._-](\h+)\.zip/i)
```

Besides the stylistic tweaks for the sake of standardization, this suggestion removes `$`, as anchoring the end with `\.zip` should be fine by itself here. The `livecheck` block looks good to me otherwise 👍

Reply:
Hah, thanks @samford.  I don't know why I reversed the `[._-]`.

Reply:
For what it's worth, I originally went with that order years ago because it puts the hyphen at the end (so it's not a character range) and `[._-]` is easier to type than `[_.-]` (with fewer trips from the top to the bottom of the keyboard). If I'm being honest, I also chose the former because it kind of looks like a winking face 😉 (conveniently the emoji has the same winking eye), so it adds a little whimsy to a super-common pattern.

I find that understanding why something is the way it is sometimes helps me to remember arbitrary things like this, so there's a bit of weird livecheck trivia for you 😄

Reply:
@samford Are you sure that's not a stickman on his back doing a leg press workout? 🤔